adambray.txt

```diff
@@ -0,0 +1,3 @@
+who is adambray
+is it a person who is mad
+he might be normal.
```

Comment:
questions that I have
why crunchwrap78
must you be supreme?

cc: @Crunchwrap78 @RobertAKARobin